apps/btshell/src/cmd.c

```diff
@@ -3540,6 +3543,24 @@ static const struct shell_cmd_help l2cap_disconnect_help = {
     .params = l2cap_disconnect_params,
 };
 
+/*****************************************************************************
+ * $l2cap-reconfig                                                           *
+ *****************************************************************************/
+
+static const struct shell_param l2cap_reconfig_params[] = {
+    {"conn", "connection handle, usage: =<UINT16>"},
+    {"mtu", "new mtu, usage: =<UINT16>, default: 0 (no change)"},
+    {"mps", "new mps, usage: =<UINT16>, default: 0 (no change)"},
+    {"idxs", "list of channel indexes, usage: idxs=01:03"},
```

Comment:
better use comma as separator - we use colon for byte streams already so it will be confusing

Reply:
ok